README.md

```diff
@@ -94,8 +94,10 @@ merged into the RFC repository as a markdown file. At that point the RFC is
 "active" and may be implemented with the goal of eventual inclusion into Ursa.
 
   - Fork the RFC repo [RFC repository]
-  - Copy `0000-template.md` to `text/0000-my-feature.md` (where "my-feature" is
-    descriptive. don't assign an RFC number yet).
+  - Copy `0000-template.md` to `0000-my-feature.md` (where "my-feature" is
+    descriptive. Don't assign an RFC number yet). 
+  - If the RFC has supporting images or diagram files, also create a folder
+    called `0000-my-feature` and put them there.
```

Comment:
Other rfc-style repos I've seen make each rfc a folder, with the primary description of the rfc as a "README.md" file within that folder. This allows for a consistent presentation of rfc's, and provides a ready location for any supporting images or files should there be any.